From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 7 May 2018 10:13:54 +0900
Subject: login: change variable type of enable_wall_messages as it matches
 Manager.enable_wall_messages

(cherry picked from commit c9482b88228db25c77ad61e119f61308af8fe2e9)
---
 src/login/logind-dbus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/login/logind-dbus.c b/src/login/logind-dbus.c
index 0a84d75..dbb21bd 100644
--- a/src/login/logind-dbus.c
+++ b/src/login/logind-dbus.c
@@ -2375,7 +2375,7 @@ static int method_set_wall_message(
         int r;
         Manager *m = userdata;
         char *wall_message;
-        int enable_wall_messages;
+        unsigned enable_wall_messages;
 
         assert(message);
         assert(m);
